src/main/java/com/redhat/exhort/integration/backend/ExhortIntegration.java

```diff
@@ -56,6 +57,16 @@ public class ExhortIntegration extends EndpointRouteBuilder {
 
   private static final String GZIP_ENCODING = "gzip";
 
+  public static final Processor excludeOrIncludeProvider =
```

Comment:
You can achieve the same using the `VulnerabilityProvider.getEnabled` method in the routes. See my suggestion down below

Reply:
Done.